### PR TITLE
New package: HeroicGamesLauncher-2.8.0

### PR DESCRIPTION
**Pull request by sirkhancision**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, x86_64-glibc

### CODE REVIEW

**Review by Bnyro — APPROVED** [review]
Built and tested successfully with Node 18.16.

**Comment by sirkhancision** [comment]
Changed from using the pre-compiled electron binaries to using the binaries from the package. Also made it use packages for legendary and gogdl.

**Review by Duncaen — CHANGES_REQUESTED** [review]
This should be split into two commits, one per new package.

File: srcpkgs/HeroicGamesLauncher/template [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+depends="electron${_electron_version} legendary heroic-gogdl"
+checkdepends="tar xz"
+short_desc="Open source launcher for GOG and Epic Games"
+maintainer="sirkhancision <jsantiago12tone@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher"
+distfiles="https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/archive/v${version}.tar.gz"
+checksum=59f068d6ba7e6c9c5d3c08f06b4cee23f07452777b9edb8baa61895fa9f40320
+nopie_files="/usr/lib/HeroicGamesLauncher/resources/app.asar.unpacked/build/bin/linux/gogdl
+/usr/lib/HeroicGamesLauncher/resources/app.asar.unpacked/build/bin/linux/legendary"
```
Comment (Duncaen) [inline srcpkgs/HeroicGamesLauncher/template:17]: Indent here with one space when splitting values in double quotes into multiple lines.

**Comment by Bnyro** [comment]
patch for 2.9.0:
```diff
diff --git a/srcpkgs/HeroicGamesLauncher/template b/srcpkgs/HeroicGamesLauncher/template
index 6bca53f7ca..e2f80bdfed 100644
--- a/srcpkgs/HeroicGamesLauncher/template
+++ b/srcpkgs/HeroicGamesLauncher/template
@@ -1,6 +1,6 @@
 # Template file for 'HeroicGamesLauncher'
 pkgname=HeroicGamesLauncher
-version=2.8.0
+version=2.9.0
 revision=1
 archs="x86_64*"
 _electron_version=24
@@ -12,9 +12,10 @@ maintainer="sirkhancision <jsantiago12tone@gmail.com>"
 license="GPL-3.0-or-later"
 homepage="https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher"
 distfiles="https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/archive/v${version}.tar.gz"
-checksum=59f068d6ba7e6c9c5d3c08f06b4cee23f07452777b9edb8baa61895fa9f40320
+checksum=b0b0e61f5f20af09527fc53b3493ede170fc7df2a148bcce7b4d83b441b8203b
 nopie_files="/usr/lib/HeroicGamesLauncher/resources/app.asar.unpacked/build/bin/linux/gogdl
-/usr/lib/HeroicGamesLauncher/resources/app.asar.unpacked/build/bin/linux/legendary"
+	/usr/lib/HeroicGamesLauncher/resources/app.asar.unpacked/build/bin/linux/legendary
+	/usr/lib/HeroicGamesLauncher/resources/app.asar.unpacked/build/bin/linux/nile"
 
 pre_build() {
 	jq 'del(.scripts.prepare)' package.json >tmp.json
```

**Comment by Bnyro** [comment]
2.9.0 doesn't seem to launch, not really sure why

**Pull request closed without merging**